health-sdk/sdk/src/main/res/values/strings.xml

```diff
@@ -52,4 +52,8 @@
     <string name="ghs_more_information_fragment_title">Mehr Informationen</string>
     <string name="ghs_privacy_policy">Datenschutzerklärung</string>
     <string name="ghs_privacy_policy_link_url"/>
+
+    //TODO get German wording and include overridable strings in documentation
```

Comment:
@a-szotyori , here are the strings for which German wording is needed. I added the `Invalid Amount` string just so that the `TalkBack` feature would speak out something more meaningful than just `Invalid`.

Reply:
Thank you, @danicretu for highlighting what needs to be translated. Here are the German translations:
* ghs_close_content_description -> "Schließen"
* ghs_invalid_amount_content_description -> "Ungültiger Bezahlbetrag"